.github/workflows/checks.yml

```
@@ -0,0 +1,24 @@
+name: Run XO and Prettier
+on:
+  pull_request:
+
+jobs:
+  run-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the application's source code
+        uses: actions/checkout@v2
+
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install application dependencies
+        run: npm ci
+
+      - name: Run Prettier
+        run: npm run checkformat
+
+      - name: Run XO script
+        run: npm run lint
```

I think the `name`s here could be slightly more descriptive:
```suggestion
      - name: Check code formatted correctly
        run: npm run checkformat

      - name: Check code conforms to linting rules
        run: npm run lint
```